_posts/2017/2017-12-31-avoid-double-negatives.md

```diff
@@ -48,13 +48,15 @@ end
 
 ## But why?
 
-Ruby’s conditional syntax is ‘truthy’. In Ruby’s case that means `nil` is equivalent to `false`. There is no need to do a negated `#nil?` check on an object as it is redundant as shown in the first two examples.
+Ruby’s conditional syntax is ‘truthy’, meaning `nil` is equivalent to `false`. Therefore a negated `#nil?` check on an object is redundant.
```

Comment:
I thin to make this whole section clearer you need to:

* explain upfront what you mean by "Ruby's conditional syntax is 'truthy'. Can you find another explanation that isn't just "meaning `nil` is equivalent to `false`." Because I feel like you need one more step in between before you get to the 'nil' bit. Does that make sense?

* tie each paragraph to one of the examples (like you had before?) but explain a bit more. For example, before you had something like: "it's redundant in the first two examples" but I think it'll be more powerful to confirm/explain what the non-redundant version looks like.

I know you're aiming for brevity here...but let's get all of the explanations clear then we can see where we can cut down.

Also, are there possibly two "One Ruby Things" in here? One around `nil` and `!!` and another around `unless`/`else` and using `if` with a positive?

Reply:
There were three.

Reply:
Ahahaha.